erpnext/hr/doctype/employee/employee.py

```diff
@@ -250,3 +257,20 @@ def deactivate_sales_person(status = None, employee = None):
 		sales_person = frappe.db.get_value("Sales Person", {"Employee": employee})
 		if sales_person:
 			frappe.db.set_value("Sales Person", sales_person, "enabled", 0)
+
+@frappe.whitelist()
+def create_user(employee, user = None):
+	user = get_mapped_doc("Employee", employee,
+		{"Employee": {
+			"doctype": "User",
+			"field_map": {
+				"employee_name": "first_name",
+				"prefered_email": "email",
+				"gender": "gender",
+				"date_of_birth": "birth_date",
+				"cell_number": "phone",
+				"bio": "bio"
+			}
+		}}, user)
+	user.save()
```

Comment:
user should be enabled?

Also, why use mapper for this? May be `new_doc` will be sufficient and cleaner in this case.